pkgs/development/python-modules/mariadb/default.nix

```diff
@@ -1,20 +1,22 @@
-{ buildPythonPackage, fetchPypi, libmysqlclient, lib, pythonOlder }:
+{ buildPythonPackage, fetchPypi, libmysqlclient, wheel, setuptools, packaging, lib, pythonOlder }:
 
 buildPythonPackage rec {
   pname = "mariadb";
-  version = "1.1.4";
+  version = "1.1.10";
   format = "setuptools";
```

Comment:
Could you please replace this line with `pyproject = true;` to avoid the legacy build?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters